Fixed fused release mode when the .love file is in love.app/Contents/Resources.
Fixed auto-detection of .love file in Contents/Resources when the love
executable binary is run directly.
https://github.com/love2d/love/commit/c730906c26a99c04cc6ad9df4bbb471847a75d17
--- platform/macosx/SDLMain.m.orig	2012-04-02 15:31:32.000000000 -0500
+++ platform/macosx/SDLMain.m	2024-05-15 04:20:09.000000000 -0500
@@ -390,6 +390,14 @@
         gArgv[1] = NULL;
         gArgc = 1;
         gFinderLaunch = YES;
+    } else {
+        int i;
+        gArgc = argc;
+        gArgv = (char **) SDL_malloc(sizeof (char *) * (argc+1));
+        for (i = 0; i <= argc; i++)
+            gArgv[i] = argv[i];
+        gFinderLaunch = NO;
+    }
 		
 		/* check to see if there are any .love files in Resources - props to stevejohnson/diordna */
 		NSArray *lovePaths = [[NSBundle mainBundle] pathsForResourcesOfType:@"love" inDirectory:nil];
@@ -418,14 +426,6 @@
 			gArgv[gArgc++] = arg;
 			gArgv[gArgc] = NULL;
 		}
-    } else {
-        int i;
-        gArgc = argc;
-        gArgv = (char **) SDL_malloc(sizeof (char *) * (argc+1));
-        for (i = 0; i <= argc; i++)
-            gArgv[i] = argv[i];
-        gFinderLaunch = NO;
-    }
 
 #if SDL_USE_NIB_FILE
     [SDLApplication poseAsClass:[NSApplication class]];
